yesod-core/test/YesodCoreTest/Exceptions.hs

```diff
@@ -41,7 +41,7 @@ case500 :: IO ()
 case500 = runner $ do
     res <- request defaultRequest
     assertStatus 500 res
-    assertBody "FOOBAR" res
+    assertBodyContains "FOOBAR" res
```

Comment:
Ah, OK, that makes sense. I'm on board with that change then. Merging, thank you for all the work on this!